examples/usage/main.tf

```
@@ -1,3 +1,34 @@
-module "terraform_module_example" {
-  source = "cloudeteer/terraform-module-example/azurerm"
+resource "azurerm_resource_group" "example" {
+  location = "germanywestcentral"
+  name     = "rg-example-dev-gwc-01"
+}
+
+resource "azurerm_virtual_network" "example" {
+  name                = "vnet-example-dev-gwc-01"
+  location            = azurerm_resource_group.example.location
+  resource_group_name = azurerm_resource_group.example.name
+
+  address_space = ["10.0.0.0/16"]
+}
+
+resource "azurerm_subnet" "example" {
+  name                = "snet-example-dev-gwc-01"
+  resource_group_name = azurerm_resource_group.example.name
+
+  address_prefixes     = ["10.0.2.0/24"]
+  virtual_network_name = azurerm_virtual_network.example.name
+}
+
+module "example" {
+  source = "cloudeteer/launchpad/azurerm"
+
+  resource_group_name = azurerm_resource_group.example.name
+  location            = azurerm_resource_group.example.location
+
+  runner_github_pat  = "github_pat_0000000000000000000000_00000000000000000000000000000000000000000000000000000000000"
+  runner_github_repo = "owner/repository"
+
+  virtual_network_address_space = azurerm_virtual_network.example.address_space
+  subnet_address_prefixes       = azurerm_subnet.example.address_prefixes
+  management_group_names        = ["mg-example"]
 }
```

These variables should not be hardcoded. Instead, they are automatically populated during runtime in the existing launchpad deployments. Therefore, the root module using the launchpad module should also rely on variables set at runtime through environment variables (`TF_VAR_*`).

```suggestion
variable "runner_github_pat" {}
variable "runner_github_repo" {}

module "example" {
  source = "cloudeteer/launchpad/azurerm"

  resource_group_name = azurerm_resource_group.example.name
  location            = azurerm_resource_group.example.location

  runner_github_pat  = var.runner_github_pat
  runner_github_repo = var.runner_github_repo

  virtual_network_address_space = azurerm_virtual_network.example.address_space
  subnet_address_prefixes       = azurerm_subnet.example.address_prefixes
  management_group_names        = ["mg-example"]
}
```